iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -231,6 +235,23 @@ + (void)setNotificationWillShowInForegroundBlock:(OSNotificationWillShowInForegr
 OSNotificationOpenedBlock notificationOpenedHandler;
 + (void)setNotificationOpenedBlock:(OSNotificationOpenedBlock)block {
     notificationOpenedHandler = block;
+    [self fireNotificationOpenedHandlerForUnprocessedEvents];
+}
+
++ (NSMutableArray<OSNotificationOpenedResult*>*)getUnprocessedOpenedNotifis {
+    if (!unprocessedOpenedNotifis)
+        unprocessedOpenedNotifis = [NSMutableArray new];
+    return unprocessedOpenedNotifis;
+}
+
++ (void)fireNotificationOpenedHandlerForUnprocessedEvents {
+    if (!notificationOpenedHandler)
+        return;
+    
+    for(OSNotificationOpenedResult* notification in [self getUnprocessedOpenedNotifis]) {
```

Comment:
space after the for 🙏

Reply:
Good catch! I also do that 🤣

Reply:
Created fixup commit 4e0fc0c2b46a1660a4d3b706c161354e85f682f2 to address this.